trade-finance-manager-api/api-tests/v1/deals/deals-submit.tasks.api-test.js

```diff
@@ -123,6 +122,8 @@ describe('/v1/deals', () => {
 
           const { email: expectedTeamEmailAddress } = MOCK_TEAMS.find((t) => t.id === firstTask.team.id);
 
+          const submissionDate = new Date(Number(body.dealSnapshot.details.submissionDate));
```

Comment:
Does this actually return a string that needs to be cast?

Reply:
submissionDate is one of the culprits from `portal-api` changes that's constructed from `Date().valueOf().toString()` or similar so should be a number but is often saved as a string.